src/Mintette/Main.hs

```diff
@@ -50,6 +42,35 @@ mainServe ctxArg Opts.ServeOptions {..} Opts.Options {..} = do
         env = M.mkRuntimeEnv cloActionLogsLimit sk
     M.launchMintetteReal cloRebuildDB epochDelta cloPort env dbPath ctxArg
 
+mainAddToBank :: M.ContextArgument -> Opts.AddToBankOptions -> Opts.Options -> IO ()
+mainAddToBank ctxArg Opts.AddToBankOptions {..} Opts.Options {..} = do
+    skEither <- try $ readSecretKey cloSecretKeyPath_
+    sk <-
+        case skEither of
+            Left (_ :: SomeException) | cloAutoCreateKey_ -> createKeypair cloSecretKeyPath_
+            Left err -> throwM err
+            Right sk -> return sk
+    M.addToBank cloRebuildDB cloPath ctxArg sk cloMintetteHost cloMintettePort
+
 mainDumpStatistics :: M.ContextArgument -> Opts.Options -> IO ()
 mainDumpStatistics ctxArg Opts.Options {..} = do
     M.dumpStorageStatistics cloRebuildDB cloPath ctxArg
+
+mainCreatePermissionKeypair :: M.ContextArgument -> Opts.Options -> IO ()
+mainCreatePermissionKeypair _ Opts.Options {..} = do
+    directory <- defaultSecretKeyPath
+    _ <- createKeypair directory
+    return ()
```

Comment:
Usually we write it as `() <$ createKeypair directory`, because it looks cool, takes 1 line and doesn't require any imports :)